src/Commands/Env/DeployCommand.php

```diff
@@ -9,11 +9,20 @@ class DeployCommand extends TerminusCommand
 {
 
     /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
```

Comment:
It's not ideal, but we don't have a pattern for accessing the collections right now so this works fine for now. All of the open PRs have a different method of doing this so we'll need to take another pass once we've decided on the one true way.